lib/iruby/command.rb

```diff
@@ -1,19 +1,34 @@
 require 'shellwords'
-require 'fileutils'
+require 'pathname'
```

Comment:
1. `pathname` provides an OO wrapper for `File`, `FileTest ` and `Dir`, which is neater and more ruby style.
2. I'm not sure whether `FileUtils` deals with windows pathname correctly, but `pathname` does.